contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -0,0 +1,487 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    attr, to_binary, Addr, Binary, Deps, DepsMut, Env, MessageInfo, Response, StdResult, SubMsg,
+    Uint128, WasmMsg,
+};
+
+use crate::state::{Config, CLAIM, CONFIG, MERKLE_ROOT, STAGE};
+
+use crate::error::ContractError;
+use crate::msg::{
+    ConfigResponse, ExecuteMsg, InstantiateMsg, IsClaimedResponse, LatestStageResponse,
+    MerkleRootResponse, MigrateMsg, QueryMsg,
+};
+use cw20::Cw20ExecuteMsg;
```

Comment:
Nit pick, I would move these `use` up higher (and run `cargo fmt` again). I like to list crate internal imports at the end